document_url/README.rst

```diff
@@ -40,7 +40,9 @@ Contributors
 ------------
 
 * Jonathan Nemry <jonathan.nemry@acsone.eu>
-* Pedro M. Baeza <pedro.baeza@tecnativa.com
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>
+* Nicolás Ramos <contacto@difusionvisual.com>
+* Antonio Cánovas <antonio.canovas@ingenieriacloud.com>
```

Comment:
Well, sometimes a "designer" can be put here, because he/she has contributed in the design of the module, but I know the history of this module (I developed it initially for him), and there's no contribution except the financial one. The line here is thin, so it's difficult to reflect in a guideline...